### PR TITLE
CDAP-12767 update default configurations

### PR DESCRIPTION
**Pull request by dereklwood**
Updates default configurations per https://issues.cask.co/browse/CDAP-12767

Related PRs:
CDAP (packages): https://github.com/caskdata/cdap/pull/9737
CSD: https://github.com/caskdata/cm_csd/pull/187
Ambari: https://github.com/caskdata/cdap-ambari-service/pull/288

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -182,18 +182,20 @@
 
   <property>
     <name>twill.java.heap.memory.ratio</name>
-    <value>0.6</value>
+    <value>0.7</value>
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: Keep this at 0.6. It applies to application containers as well.

File: cdap-common/src/main/resources/cdap-default.xml [hunk 2]
```diff
@@ -1684,9 +1684,10 @@
 
   <property>
     <name>master.service.memory.mb</name>
-    <value>512</value>
+    <value>1024</value>
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml:1687]: Why need this change?

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (dereklwood) [reply cdap-common/src/main/resources/cdap-default.xml]: this is per the jira: https://issues.cask.co/browse/CDAP-12767, and to keep configs consistent across CM/Ambari/Packages.  I guess an alternative is to instead specify all ``[service].twill.java.heap.memory.ratio`` also in cdap-default.xml...

File: cdap-common/src/main/resources/cdap-default.xml (hunk 2 above)
Comment (dereklwood) [reply cdap-common/src/main/resources/cdap-default.xml:1687]: this is per the jira: https://issues.cask.co/browse/CDAP-12767, and to keep configs consistent across CM/Ambari/Packages.  I believe it was in response to seeing the messaging container fail to startup.  We can discuss

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (dereklwood) [reply cdap-common/src/main/resources/cdap-default.xml]: reverting to 0.6... there was a typo in the jira.

File: cdap-docs/admin-manual/build.sh [hunk 3]
```diff
@@ -20,7 +20,7 @@ source ../vars
 source ../_common/common-build.sh
 
 DEFAULT_XML="../../cdap-common/src/main/resources/cdap-default.xml"
-DEFAULT_XML_MD5_HASH="e42bac30d3bec69fc4bf3b99e4f54271"
+DEFAULT_XML_MD5_HASH="cfde024ece2f693976ecfe707ee32724e4b9bb42"
```
Comment (chtyim) [inline cdap-docs/admin-manual/build.sh]: are you sure this is MD5? It shouldn't be longer.

**Comment by chtyim** [comment]
LGTM

**Pull request merged**